torch/_torch_docs.py

```diff
@@ -5376,6 +5376,28 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.sgn,
+           r"""
+sgn(input, out=None) -> Tensor
+
+Returns a new complex tensor with the same angle as that of the elements of :attr:`input` and absolute value 1.
```

Comment:
Probably want to reference the angle and absolute value functions

Reply:
Oh wait... we don't have complex abs yet, do we? Then you may need to define what you mean by "absolute value"

Reply:
yeah good idea. Btw both `abs` and `angle` are defined for complex tensors.

Reply:
Reading over this I kind of wonder why we didn't call this `normalize` or `hat` since that's exactly what we're doing: we're treating the complex numbers as vectors and normalizing them into unit vectors. (OK, I do know why we're calling it sgn, it's because it's the Matlab sgn thing. But maybe in the docs it would be worth mentioning this.)

Reply:
What does this function do when you have zero complex numbers in the input?

Reply:
it returns 0 (following https://en.wikipedia.org/wiki/Sign_function#Complex_signum)